### ISSUE TITLE
Node v21 marked as unsupported but it works without any problems

### ISSUE THREAD

**Issue by tirli** [effector/eslint-plugin#154]
I'm constantly getting this warning about using node v21 with this plugin, but it works perfectly fine for me. Could you add it to the engines list? Or I can create a PR. 

```
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'eslint-plugin-effector@0.13.0',
npm WARN EBADENGINE   required: { node: '^16 || ^17 || ^18 || ^19 || ^20' },
npm WARN EBADENGINE   current: { node: 'v21.6.1', npm: '10.2.4' }
npm WARN EBADENGINE }
```

**Comment by igorkamyshev** [comment]
Hello!

Basically, I was quite cautious about supporting odd versions of Node.js. However, we started to support in a couple of versions ago, so I think we can support 21 as well.

It has to be added to `package.json` as supported version and to CI matrix because we have to run tests on the new version of Node.js.

PR is wellcome, or I can do it in a week or so.

**Comment by igorkamyshev** [comment]
Done ✅ https://github.com/effector/eslint-plugin/commit/21f4ea68729b9f781b533e963b1370444aca46d1

Will be released with other changes soon 🔜
